_security-analytics/sec-analytics-config/log-types.md

```diff
@@ -1,51 +1,79 @@
+- **Network Activity** includes [DNS]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/dns/), [Network]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/network/), [NetFlow]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/netflow/) and [VPC Flow]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/vpc/).
+- **Security** includes [WAF]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/waf/).
+- **System Activity** includes [Linux]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/linux/) and [Windows]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/windows/).
+- **Other** accounts for log types that are not under a specific category. For more information, refer to [Other log types]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/other/).
+
+
+## Creating custom log types
+
+When connecting to a data source not supported by the standard log type, create a custom log type by following these steps: 
+
+1. From the OpenSearch Dashboards, select **OpenSearch Plugins** > **Security Analytics**, and then select **Detectors** > **Log types**.
+1. Select **Create log type**.
+1. Enter a name and, optionally, a description for the log type.
+   
+   The log type name supports characters a--z (lowercase), 0--9, hyphens, and underscores.
+   {: .note }
+   
+1. Select a category. The categories are listed in [Supported log types]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/log-types/).
+1. Select **Create log type** in the lower-right corner of the screen. The screen returns to the **Log types** page. The new log type appears in the list. Note that the source for the new log type indicates **Custom**.
+
+## About field mappings
+
+The log type specified when creating detectors determines which fields are available for mapping. For example, when **Windows logs** is selected, this parameter and the specific detection rules determine the list of detection field names available for the mapping. Similarly, the selected data source determines the list of log source field names that are available for the mapping.
+
+Security Analytics uses prepackaged Sigma rules for detector creation. It can automatically map important fields of a particular log type with the relevant fields in Sigma rules. In the **Field Mappings** section for [detector creation](<insert-missing-link>) is a view of fields that have been mapped automatically. From this view, you can customize, change, or add new field mappings. When a detector includes customized rules, you can manually map detector rule field names to log source field names.
```

Comment:
```suggestion
Security Analytics uses prepackaged Sigma rules for detector creation. It can automatically map important fields of a particular log type with the relevant fields in Sigma rules. In the **Field Mappings** section for detector creation is a view of fields that have been mapped automatically. From this view, you can customize, change, or add new field mappings. When a detector includes customized rules, you can manually map detector rule field names to log source field names.
```